packages/ui/src/Button.tsx

```diff
@@ -183,7 +188,7 @@ export const Button = forwardRef<HTMLElement, ButtonProps>(
           >
             <span data-test="button-outline" className={cn(styles.outline, { [styles.inset]: outline === 'inset' }, outlineClassName)} />
             <span className={cn(styles.body, bodyClassName)} ref={mergeRefs([ref, tooltipRef])}>
-              {loading && <Loader className={styles.iconLeft} />}
+              {loading && <Loader className={styles.iconLeft} size={size === 'small' ? 'small' : 'medium'} />}
```

Comment:
nit: Could we just go like this, since we are declaring the type as a union above `'medium' | 'small'` and then make a fallback `size = 'medium',`

```suggestion
              {loading && <Loader className={styles.iconLeft} size={size} />}
```

Reply:
Good catch, that's a silly mistake :smile:

Reply:
Changed in https://github.com/hazelcast/hive/pull/279/commits/4e307d6d8048f2918e52ce3a1796230bd17efb77